CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # SPDX-License-Identifier: Apache-2.0
 
-# Fabric Maintainers
-*	@ale-linux @binhn @c0rwin @christo4ferris @denyeart @guoger @JonathanLevi @jyellick @kchristidis @manish-sethi @mastersingh24 @muralisrini @sykesm @yacovm
+# Fabric Release Maintainers
+*	@hyperledger/fabric-release-maintainers
```

Comment:
I believe this should be the core fabric maintainers and not just release maintainers. Looking through the groups, it doesn't look like we have one for the team; instead the fabric-maintainers group has nested groups for the various sub-projects. (I may be missing something, however...)